jsettlers.logic/src/jsettlers/logic/buildings/Building.java

```diff
@@ -715,4 +704,13 @@ public short getPartitionId() {
 	public boolean cannotWork() {
 		return false;
 	}
+
+	public enum EBuildingState {
```

Comment:
Can you please add a comment here that the order is important and should not be changed - otherwise savegames will break.